### PR TITLE
Hello Matlab

### PR DESCRIPTION
**Pull request by Akashsnar**
Created Hello world code in Matlab.

### CODE REVIEW

File: hello.m [hunk 1]
```diff
@@ -0,0 +1,2 @@
+x=input('', 's');
+y="hello "+x+"!"
```
Comment (LordUbuntu) [inline hello.m]: Please put this all into a `main` function.

File: hello.m [hunk 2]
```diff
@@ -0,0 +1,2 @@
+x=input('', 's');
```
Comment (LordUbuntu) [inline hello.m]: please change `x` to `name`

**Comment by LordUbuntu** [comment]
No activity, and user account is sus.

**Comment by Akashsnar** [comment]
sir, why you label my repository spam, when it follow all conditions.

**Comment by LordUbuntu** [comment]
@Akashsnar 
See [here](https://github.com/LordUbuntu/hello3000#contributing)

> Subject to the following limitations:
> read input and assign to variable name, write output "Hello [name]!"
> the program should be contained in a main function and that function called, if it can be done in the language

But your activity is encouraging! Make these changes and I'll merge your PR:
- put all the program into a main function (that should be an option in matlab)
- rename `x` to `name`
- use a print function like `disp` or `fprintf` instead of assignment to `y`

**Review by LordUbuntu — APPROVED** [review]
I want to personally thank you for taking the time to make your changes, it means a lot to me. This is a very beautiful moment. Thank you, your PR has been gladly merged!

**Comment by LordUbuntu** [comment]
Truly a gorgeous submission. Thank you so very much @Akashsnar

**Pull request merged**